flink-runtime/src/main/java/org/apache/flink/runtime/state/KeyGroupsStateHandle.java

```diff
@@ -91,10 +98,10 @@ public KeyGroupsStateHandle getKeyGroupIntersection(KeyGroupRange keyGroupRange)
 
 	/**
 	 *
-	 * @return the internal key-group range to offsets metadata
+	 * @return the start key group in the key-group range of this handle
 	 */
-	public KeyGroupRangeOffsets getGroupRangeOffsets() {
-		return groupRangeOffsets;
+	public int getStartKeyGroup() {
```

Comment:
I have removed all pass-through methods except `getGroupRangeOffsets()` because `StateInitializationContextImpl$KeyGroupStreamIterator` is using it to get the iterator for key groups and their offsets.